docs/detections/alerts-run-osquery.asciidoc

```diff
@@ -35,8 +36,18 @@ image::images/setup-query.png[width=80%][height=80%][Shows how to set up the que
 TIP: To save the query for future use, click *Save for later* and define the ID,
 description, and other {kibana-ref}/osquery.html#osquery-manage-query[details].
 
-. Review the results in the table. You can also navigate to *Discover* to dive deeper into the response,
-or use the drag-and-drop *Lens* editor to create visualizations.
+. Review the results in the table. You can also:
+** Navigate to *Discover* to dive deeper into the response.
+** Use the drag-and-drop *Lens* editor to create visualizations.
+** Click the *Timeline* button (image:images/timeline-button-osquery.png[Click markdown icon,20,20]) to investigate a single query result in Timeline or *Add to timeline investigation* to investigate all results.
```

Comment:
Oh yeah, thanks for correcting the alt text. I tried separating the two options and giving them their own line, but thought it might be better to keep them together since they're similar. 

Somewhat unrelated, the Timeline button is also named **Add to timeline investigation** @tomsonpl . Do you think this should be changed in the next release? I'm not sure if it's even possible, but wanted to point it out because it might be a little confusing for users to have two actions identically named actions that do different things. 
<img width="696" alt="Screen Shot 2022-06-22 at 5 05 53 PM" src="https://user-images.githubusercontent.com/79124755/175136460-890d821a-4f55-4666-a9ad-5c3ae89793b7.png">